client/src/apiClient/QuoteClient.js

```diff
@@ -0,0 +1,20 @@
+import AbstractClient from "./AbstractClient";
+
+class QuoteClient extends AbstractClient {
+  async getRandomQuote() {
+    const resp = await QuoteClient.client()
+      .get("/quotes").catch((error) => {
+        const errorStatus = error?.response?.status;
+        if ([401, 422].includes(errorStatus)) {
+          return error.response.data;
+        }
+        throw error;
+      })
+
+    if (resp && resp.status === 201) {
```

Comment:
when we do a `GET` we should expect a 200 response, not a 201 - 201 means "created"